testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/InfinispanContainer.java

```diff
@@ -50,6 +54,16 @@ public InfinispanContainer() {
         withEnv("USER", USERNAME);
         withEnv("PASS", PASSWORD);
         withNetworkMode("host");
+        
+        Path dir = Path.of(Path.of("").toAbsolutePath() + "/target/lib");
+        Path timeTaskPath;
+        try {
+            timeTaskPath  = Files.find(dir, 1, (path, attr) -> path.toString().endsWith(".jar")).findFirst().orElse(null);
```

Comment:
This assumes that there is only a single jar. Can there be multiple .jar files?

Reply:
In theory, it can. So I used more unique suffix `"integration-arquillian-testsuite-providers-" + projectVersion + ".jar"` which should be unique.